src/App.js

```diff
@@ -213,12 +212,8 @@ function LoadedWalletComponent({ children }) {
 
   // Check version
   if (isVersionAllowed === undefined) {
-    throw new Error('[LoadedWalletComponent] isVersionAllowed is undefined');
-    // return <Redirect to={{
-    //   pathname: '/loading_addresses/',
-    //   state: {path: match.url},
-    //   waitVersionCheck: true
-    // }} />;
+    // The version information is not yet available, no op
+    return;
```

Comment:
Why this change?

Reply:
Because when the `ServerChange` sets the `isVersionAllowed` to `undefined` to force a revalidation of the version, the `LoadedWalletComponent` would throw an exception and the whole application would come to a halt.

Instead, it should just wait a few more fractions of a second for this property to be populated and the app can continue working normally.